AllNews.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react'
+import './covid.scss'
+import ig4 from './images/ig4.jpg'
+import ig5 from './images/ig5.jpg'
+import ig6 from './images/ig6.jpg'
+import ig7 from './images/ig7.jpg'
+
+
+class AllNews extends Component {
+    render() {
+        return (
+            <div>
+                <marquee width = "65%" 	
+scrollamount="13"><h1 className='head3'>HERE ARE SOME NEWS CHANNELS TO FOLLOW LIVE COVID UPDATES IN INDIA</h1></marquee> 
+                <ul className='mycard'>
+
+                    <li>
+                        <div className='card-3'>
+                            <div className='card-3-main'>
+                                <div className='card-3-head'>
+                                    <a href='https://economictimes.indiatimes.com/coronavirus'><img src={ig4} className="my_ig4" /></a>
+
+                                </div>
+                                <div className='card-3-body'>
+                                    <h3>THE ECONOMIC TIMES</h3>
+                                    <h5>Since 1961</h5>
+                                </div>
+                            </div>
+
+                        </div>
+                    </li>
+                    <li>
+                        <div className='card-3'>
+                            <div className='card-3-main'>

```

Comment:
why do we need these many Div sections?

Reply:
rather than displaying these many static <li.> tags you can create array of objects with properties and loop it